com.unity.robotics.urdf-importer/Runtime/MeshProcessing/StlAssetPostProcessor.cs

```diff
@@ -26,14 +26,27 @@ public class StlAssetPostProcessor : AssetPostprocessor
     public class StlAssetPostProcessor
 #endif    
     {
-        private static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromPath)
+        static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromPath)
         {
 #if UNITY_EDITOR
             if (!RuntimeURDF.IsRuntimeMode())
             {
-                foreach (string stlFile in importedAssets.Where(x => x.ToLowerInvariant().EndsWith(".stl")))
+                foreach (var stlFile in importedAssets.Where(x => x.ToLowerInvariant().EndsWith(".stl")))
                 {
-                    createStlPrefab(stlFile);
+                    if (stlFile.StartsWith("Assets"))
```

Comment:
I recommend we also handle directory name with different casing: i.e. `stlFile.ToLower().StartsWith("assets")`